libs/core/src/scss/components/_grid.scss

```diff
@@ -0,0 +1,97 @@
+@use 'sass:map';
+
+/* stylelint-disable custom-property-pattern */
+$breakpoint-map: (
+  'xs': 0,
+  'sm': 632px,
+  'md': 768px,
+  'lg': 992px,
+  'xl': 1200px,
+);
+
+.kirby-grid {
+  --kirby-grid-columns: 12;
+  --kirby-grid-columnSpacing: 24px;
+  --kirby-grid-rowSpacing: 24px;
+
+  min-width: 0;
```

Comment:
Why `min-width: 0` here? Is that not the initial value?

Reply:
Nope you are right, it was only needed on the items not the container :-)